site/content/docs/commands/env-package.en.md

```diff
@@ -0,0 +1,30 @@
+# env package
+```console
+$ copilot env package [flags]
+```
+
+## What does it do?
+`copilot env package` prints the CloudFormation stack template and configuration used to deploy an environment.
+
+## What are the flags?
+```console
+-a, --app string          Name of the application.
+  --force               Optional. Force update the environment stack template.
```

Comment:
Do they align alright when rendered into the web page?

Reply:
Done!